test/v2-migration-tests/src/test/java/software/amazon/awssdk/v2migrationtests/MavenProjectTest.java

```diff
@@ -69,16 +67,17 @@ private static void deleteTempDirectories() throws IOException {
     }
 
     @Test
-    // @EnabledIf("versionAvailable")
-    @Disabled("Test is flaky")
+    @EnabledIf("versionAvailable")
     void mavenProject_shouldConvert() throws IOException {
         verifyTransformation();
         verifyCompilation();
     }
 
     private static void verifyTransformation() throws IOException {
         List<String> rewriteArgs = new ArrayList<>();
-        addAll(rewriteArgs, "mvn", "org.openrewrite.maven:rewrite-maven-plugin:run",
+        // pin version since updates have broken tests
+        String rewriteMavenPluginVersion = "5.43.0";
+        addAll(rewriteArgs, "mvn", "org.openrewrite.maven:rewrite-maven-plugin:" + rewriteMavenPluginVersion + ":run",
```

Comment:
Do we need the separate variable?

Reply:
just so easier to see and update version in future, to pin to more recent version once it works. wdyt?

Reply:
Seems like you could just as easily update the version in the if were all a single string. I don't feel strongly either way though.